keps/sig-cloud-provider/20180731-cloud-provider-docs.md

```diff
@@ -0,0 +1,143 @@
+│           ├── kubelet.manifest                   # an example manifest of kubelet using the in-tree integration of this cloud provider
+│       └── out-of-tree/
+│           ├── apiserver.manifest                 # an example manifest of apiserver using the out-of-tree integration of this cloud provider
+│           ├── kube-controller-manager.manifest   # an example manifest of kube-controller-manager using the out-of-tree integration of this cloud provider
+│           ├── cloud-controller-manager.manifest  # an example manifest of cloud-controller-manager using the out-of-tree integration of this cloud provider
+│           ├── kubelet.manifest                   # an example manifest of kubelet using out-of-tree integration of this cloud provider
+```
+
+#### Requirement 2: Resource Management
+
+List the latest annotations and labels that are cloud-provider dependent and can be used by the Kubernetes administrator. The contents of these files should be kept up-to-date as annotations/labels are deprecated/removed/added/updated. Labels and annotations should be grouped based on the resource they are applied to. For example the label `beta.kubernetes.io/instance-type` is applied to nodes so it should be added to `k8s.io/cloud-provider-foobar/docs/node/labels.md`.
+
+```
+cloud-provider-foobar/
+├── ...
+├── ...
+├── docs
+│   └── resources
+│       └── node/
+│           ├── labels.md        # outlines what annotations that can be used on a Node resource
+│           ├── annotations.md   # outlines what annotations that can be used on a Node resource
+│           ├── README.md        # outlines any other cloud provider specific details worth mentioning regarding Nodes
+│       └── service/
+│           ├── labels.md        # outlines what annotations that can be used on a Service resource
+│           ├── annotations.md   # outlines what annotations that can be used on a Service resource
+│           ├── README.md        # outlines any other cloud provider specific details worth mentioning regarding Services
+│       └── persistentvolumes/
+│           ├── ...
+│       └── ...
+│       └── ...
+```
+
+### Implementation Details/Notes/Constraints [optional]
+
+The requirements above lists the bare minimum documentation that any cloud provider in the Kubernetes ecosystem should have. Cloud providers may choose to add more contents under the `docs/` directory as they see fit.
+
+### Risks and Mitigations
+
+There are no risks and mitigation with respect to adding documentation for cloud providers. If there are any, they would already exist in the various places the cloud provider docs exists today and implementing this KEP would not increase those risks.
+
+### Graduation Criteria
+
+* All cloud providers have written docs that adhere to the format specified in this KEP.
+* SIG Docs is consuming docs written by each provider in a way that is easily consumable for Kubernetes' users.
```

Comment:
In addition to standardizing content in this template, it may be worthwhile to start thinking about how to standardize presentation of linked content. The question is definitely out of scope for this PR, and it's a downstream topic for SIG Docs to think about. I just want to raise visibility about not only standardizing cloud provider content, but standardizing how we present links.